rust-toolchain.toml

```diff
@@ -1,6 +1,5 @@
 [toolchain]
 channel = "nightly-2023-04-25"
-targets = [
-  "wasm32-unknown-unknown",
-]
+components = [ "rustfmt", "clippy", "llvm-tools" ]
```

Comment:
We can remove `profile` if we set `components` I guess. Or not to set `rustfmt` and `clippy`.

https://rust-lang.github.io/rustup/concepts/profiles.html